<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetSelect</class>
 <widget class="QWidget" name="WidgetSelect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>402</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Available &amp;widgets</string>
       </property>
       <property name="buddy">
        <cstring>widgetList</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="widgetList"/>
     </item>
     <item>
      <widget class="QPushButton" name="addWidget">
       <property name="toolTip">
        <string>Adds widget to list of selected widgets</string>
       </property>
       <property name="text">
        <string>&amp;Add</string>
       </property>
       <property name="icon">
        <iconset resource="../../../resource/default.qrc">
         <normaloff>:/icons/default/icons/add.png</normaloff>:/icons/default/icons/add.png</iconset>
       </property>
       <property name="shortcut">
        <string>Ctrl+Return</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="widgetDescLabel">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>&amp;Selected widgets:</string>
         </property>
         <property name="buddy">
          <cstring>finalList</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListView" name="finalList">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="movement">
          <enum>QListView::Free</enum>
         </property>
         <property name="isWrapping" stdset="0">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="delWidget">
         <property name="toolTip">
          <string>Remove selected widget</string>
         </property>
         <property name="text">
          <string>&amp;Delete</string>
         </property>
         <property name="icon">
          <iconset resource="../../../resource/default.qrc">
           <normaloff>:/icons/default/icons/cancel.png</normaloff>:/icons/default/icons/cancel.png</iconset>
         </property>
         <property name="shortcut">
          <string>Backspace</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="upButton">
         <property name="toolTip">
          <string>Moves up the selected widget</string>
         </property>
         <property name="icon">
          <iconset resource="../../../resource/default.qrc">
           <normaloff>:/icons/default/icons/arrow_up.png</normaloff>:/icons/default/icons/arrow_up.png</iconset>
         </property>
         <property name="shortcut">
          <string>Ctrl+Up</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="downButton">
         <property name="toolTip">
          <string>Moves down the selected widget</string>
         </property>
         <property name="icon">
          <iconset resource="../../../resource/default.qrc">
           <normaloff>:/icons/default/icons/arrow_down.png</normaloff>:/icons/default/icons/arrow_down.png</iconset>
         </property>
         <property name="shortcut">
          <string>Ctrl+Down</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>widgetList</tabstop>
  <tabstop>addWidget</tabstop>
  <tabstop>finalList</tabstop>
  <tabstop>upButton</tabstop>
  <tabstop>downButton</tabstop>
  <tabstop>delWidget</tabstop>
 </tabstops>
 <resources>
  <include location="../../../resource/default.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>addWidget</sender>
   <signal>clicked()</signal>
   <receiver>widgetList</receiver>
   <slot>setFocus()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>148</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
